boston/README.md

```diff
@@ -123,19 +122,18 @@ python -m trainer.task \
  --job-dir=$JOB_DIR
 ```
 
-* **Google Cloud ML Engine:**
+* **Google Cloud ML Engine**
 
 * **GCloud configuration:**
 
 ```
-export BUCKET_NAME=your-bucket-name
 DATE=`date '+%Y%m%d_%H%M%S'`
 export JOB_NAME="boston_keras_$DATE"
-export JOB_DIR=gs://$BUCKET_NAME/$JOB_NAME
-export REGION=us-central1
+export JOB_DIR=boston_$DATE
+rm -rf $JOB_DIR
 export TRAIN_FILE=gs://cloud-samples-data/ml-engine/boston/boston_housing.npz
 ```
-* **Run locally:**
+* **Run locally via the gcloud emulator for Google Cloud ML Engine:**
```

Comment:
Not sure if it is really an emulator.  I think it just tries to run the package with `python -m`.